netbox_data_import/templates/netbox_data_import/import_preview.html

```
@@ -884,38 +942,55 @@ <h5 class="modal-title" id="contactCandidateModalLabel">
+            </div>
+            <div class="col-12">
+              <label class="form-label" for="contactCandidateExisting">Existing NetBox Contact</label>
+              <select class="form-select" id="contactCandidateExisting"
+                      placeholder="Type a name, email address, or phone number"></select>
+              <div class="form-text">Selecting a Contact copies its current details into the fields below.</div>
+            </div>
             <div class="col-md-4">
               <label class="form-label" for="contactCandidateName">Contact name</label>
-              <select class="form-select" id="contactCandidateName" required></select>
+              <select class="form-select" id="contactCandidateName"></select>
+              <input type="text" class="form-control mt-2" id="contactCandidateNameValue"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)